.github/workflows/ax.yml

```diff
@@ -126,10 +120,9 @@ jobs:
     strategy:
       matrix:
         config:
-          #@note llvm10 never got its own brew formula...
+          #@note llvm15 never got its own brew formula...
           # Last macos runner befor M1 (macos-14)
-          - { runner: 'macos-13', cxx: 'clang++', build: 'Release', llvm: '12' }
-          - { runner: 'macos-13', cxx: 'clang++', build: 'Release', llvm: '13' }
+          - { runner: 'macos-13', cxx: 'clang++', build: 'Release', llvm: '15' }
```

Comment:
You remove llvm14 from the weekly workflow - add it back here? Also add 13 back if our minimum is 13

Reply:
Yep, I just added 13 back in (we do also test 14 via Windows currently).